docs/proxy/google_serp_proxy/examples.md

```diff
@@ -0,0 +1,512 @@
+query = parse.urlencode({ 'q': 'Apple iPhone XS 64GB', 'tbm': 'shop' })
+url = (
+    'http://www.google.co.uk/search?%s' %
+    (query)
+)
+print(opener.open(url).read())
+</marked-tab>
+
+
+<marked-tab header="PHP" lang="php">
+<?php
+$query = urlencode('Apple iPhone XS 64GB');
+$curl = curl_init('http://www.google.co.uk/search?tbm=shop&q=' . $query);
+curl_setopt($curl, CURLOPT_RETURNTRANSFER, 1);
+curl_setopt($curl, CURLOPT_PROXY, 'http://proxy.apify.com:8000');
+// Replace <YOUR_PROXY_PASSWORD> below with your password
+// found at https://my.apify.com/proxy
+curl_setopt($curl, CURLOPT_PROXYUSERPWD, 'groups-GOOGLE_SERP:<YOUR_PROXY_PASSWORD>');
+$response = curl_exec($curl);
+curl_close($curl);
+echo $response;
+?>
+</marked-tab>
+```
+
+## [](#using-the-apify-sdk) Using the Apify SDK
+
+If you're developing an actor using the [Apify SDK](https://sdk.apify.com), you can use Apify Proxy in:
+
+* [PuppeteerCrawler](https://sdk.apify.com/docs/api/puppeteer-crawler#docsNav) using the [createProxyConfiguration()](https://sdk.apify.com/docs/api/apify#apifycreateproxyconfigurationproxyconfigurationoptions) function.
+* [requestAsBrowser()](https://sdk.apify.com/docs/api/utils#utilsrequestasbrowseroptions) function by specifying proxy configuration in the options.
+* [launchPuppeteer()](https://sdk.apify.com/docs/typedefs/launch-puppeteer#docsNav) by specifying the configuration in the function's options.
+
+### [](#get-and-parse-a-list-of-search-results) Get and parse a list of search results
+
+Get a list of search results from the USA (**google.com**) for the keyword **wikipedia** and parse them using [cheerio](https://cheerio.js.org/).
+
+```marked-tabs
+<marked-tab header="PuppeteerCrawler" lang="javascript">
+const Apify = require('apify');
+
+Apify.main(async() => {
+    const proxyConfiguration = await Apify.createProxyConfiguration({
+        groups: ['GOOGLE_SERP'],
+    });
+    const url = 'http://www.google.com/search?q=wikipedia';
+
+    const requestList = await Apify.openRequestList('my-list', [url]);
+
+    const crawler = new Apify.PuppeteerCrawler({
+        requestList,
+        proxyConfiguration,
+        gotoFunction: async({ page, request }) => {
+            await page.setRequestInterception(true);
+            page.on('request', request => {
+                if (request.resourceType() !== 'document') return request.abort();
+                return request.continue();
+            });
+            return page.goto(url);
+        },
+        handlePageFunction: async({ page, request, proxyInfo }) => {
+            await Apify.utils.puppeteer.injectJQuery(page);
+            const searchResults = await page.evaluate(() => {
+                return $('#search div.g').map(function () {
+                    return {
+                        title: $($(this).find('h3')[0]).text(),
+                        description: $(this).find('.s div .st').text(),
+                    }
+                }).toArray();
+            });
```

Comment:
I would get rid of the parsing entirely. Google changes selectors quite often, plus IMO it adds unneeded complexity to the example.